dali/test/dali_test_single_op.h

```diff
@@ -34,6 +34,12 @@ namespace dali {
 
 #define SAVE_TMP_IMAGES 0
 
+#if SAVE_TMP_IMAGES
+namespace {
+  int tmp_img_idx = 0;
```

Comment:
global indexing - avoid overwriting results from different template instance

Reply:
Do we consider any thread safety here?